Doc/faq/design.rst

```diff
@@ -528,9 +528,9 @@ Some unacceptable solutions that have been proposed:
      mydict = {[1, 2]: '12'}
      print(mydict[[1, 2]])
 
-  would raise a KeyError exception because the id of the ``[1, 2]`` used in the
-  second line differs from that in the first line.  In other words, dictionary
-  keys should be compared using ``==``, not using :keyword:`is`.
+  would raise a :exc:`KeyError` exception because the id of the ``[1, 2]`` used
```

Comment:
Please leave "in the" on the same line to reduce the number of modified lines in your change. It's fine if the line is "too long".